lib/rucio/client/uploadclient.py

```diff
@@ -62,17 +62,13 @@
 
 class UploadClient:
 
-    def __init__(self, _client=None, logger=None, tracing=True):
+    def __init__(self, _client=None, logger=logging.log, tracing=True):
         """
         Initialises the basic settings for an UploadClient object
 
         :param _client:     - Optional: rucio.client.client.Client object. If None, a new object will be created.
         :param logger:      - logging.Logger object to use for uploads. If None nothing will be logged.
```

Comment:
just saw this doc string here, should be updated wherever present and probably added when absent